src/sentry/testutils/cases.py

```diff
@@ -377,6 +387,34 @@ def assertWriteQueries(self, queries, debug=False, *args, **kwargs):
         with context:
             func(*args, **kwargs)
 
+    def create_organization(self):
```

Comment:
Turns out there already was a fixture for this (in ./fixtures). I'll remove this.